CODEOWNERS

```diff
@@ -18,3 +18,5 @@
 /OracleWebCenterSites/    @prshshuk
 /OracleWebLogic/          @mriccell
 /OracleSOASuite/          @rdas0405
+/OracleUnifiedDirectory/  @kuldeepbshah @surya902
+/OracleUnifiedDirectory/  @kuldeepbshah @surya902
```

Comment:
You have two identical lines. The second one should be `/OracleUnifiedDirectorySM/`

Reply:
This is the last remaining fix that needs to occur before I can merge this PR.